docs/other-api/create-remix.md

```diff
@@ -33,11 +33,11 @@ npx create-remix@latest --help
 `create-remix` can also be invoked using various package managers, allowing you to choose between npm, Yarn, pnpm, and Bun for managing the install process.
 
 ```sh
-npm create remix@latest <projectDir>
+npm create-remix@latest <projectDir>
```

Comment:
The existing docs are correct, making use of `npm create`, which is an alias for `npm init`: https://docs.npmjs.com/cli/v10/commands/npm-init